assignment-2/constants.rb

```diff
@@ -0,0 +1,9 @@
+module MyConstants
```

Comment:
Never use Myclass, MyConstants, etc as a class name when working on enterprise projects.
Change it to Constants